databases/week2/Homework/Homework_tables.sql

```diff
@@ -0,0 +1,152 @@
+USE Clinic_scheduling;
+
+CREATE TABLE
+  Clinics (
+    clinic_id INT NOT NULL AUTO_INCREMENT,
+    Clinic_name VARCHAR NOT NULL,
+    Clinic_address VARCHAR NOT NULL,
+    PRIMARY KEY (clinic_id)
+  );
+
+-- a USER can either have work schedule for certain date or a regular working schedule for certain weekdays
+-- hence different kind of time-slots
+CREATE TABLE
+  Working_hours (
+    date DATE,
+    weekday CHAR(3),
+    begins DATETIME NOT NULL,
+    ends DATETIME NOT NULL,
+    slot_id INT NOT NULL AUTO_INCREMENT,
```

Comment:
Usually the Primary key is the first column. It is easier to read and understand

Reply:
Changed the sequence. Thank you for noting.